### ISSUE TITLE
ERROR: CSS rules were not executed. Error when running: 'node -v'. Is Node.js available during analysis? org.sonarsource.nodejs.NodeCommandException: Error when running: 'node -v'. Is Node.js available during analysis? 	at org.sonarsource.nodejs.NodeCommand.start(NodeCommand.java:83)

### ISSUE THREAD

**Issue by hom7** [SonarSource/SonarJS#1975]
**I want to request a feature.**

**I want to report a bug.**

"ERROR: CSS rules were not executed. Error when running: 'node -v'. Is Node.js available during analysis?
org.sonarsource.nodejs.NodeCommandException: Error when running: 'node -v'. Is Node.js available during analysis?
at org.sonarsource.nodejs.NodeCommand.start (NodeCommand.java:83) "
**SonarJS version:** <br>
**SonarQube version:** <br>
docker sonarqube Version 8.2 
**SonarLint version:** <br>

**Rule key:** 

**Reproducer** 
```
// minimal reproducer if relevant
console.log("Hello, world"); // False-positive here
```

**Logs**
```
// logs of analysis if relevant
```

**Expected behavior**

Using jenkins to analyze sonarqube and analyze node.js errors can help me solve it

**I want to provide the following feedback.**

**Rule key:** 

**Example**
